src/syft/lib/python/dict.py

```diff
@@ -36,7 +36,7 @@ class Dict(UserDict, PyPrimitive):
     # python 3.8 signature includes a new PEP 570 (args, /, kwargs) syntax:
     # https://www.python.org/dev/peps/pep-0570/
 
-    def __init__(*args: Any, **kwargs: Any) -> None:
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
```

Comment:
This is also a super special case where we actually need the self as part of the args to support older python versions.